darts/tests/models/forecasting/test_dlinear_nlinear.py

```diff
@@ -338,8 +339,92 @@ def test_optional_static_covariates(self):
                     output_chunk_length=6,
                     use_static_covariates=False,
                     n_epochs=1,
-                    **tfm_kwargs
+                    **tfm_kwargs,
                 )
                 model.fit(series.with_static_covariates(None))
                 preds = model.predict(n=2, series=series)
                 assert preds.static_covariates.equals(series.static_covariates)
+
+        @pytest.mark.parametrize(
+            "config", itertools.product([3, 7, 10], [NLinearModel, DLinearModel])
+        )
+        def test_output_shift(self, config):
```

Comment:
Since this test is pretty much copy-pasted in every torch forecasting model unit-tests, would probably be better to move it to `test_torch_forecasting_model.py`?

Also, we should also add a short test to check that `historical_forecasts()` returns a ts with the expected time index (same length - assuming covariates are long enough-, but shifted)

Reply:
True!